.github/workflows/label_and_milesone_checker.yml

```diff
@@ -15,7 +15,7 @@ on:
 jobs:
   check_labels:
     if: (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ready to merge'))
-    name: Check PR has type label
+    name: Check PR is marked ready-to-merge
```

Comment:
This change is incorrect. The check is below:

 ```
      - name: Check labels
        uses: docker://agilepathway/pull-request-label-checker:latest
        with:
          any_of: bugfix,feature,documentation,performance,enhancement,maintenance
          repo_token: ${{ secrets.GITHUB_TOKEN }}
```

This *only runs* if the PR is marked ready-to-merge. But what it checks is that the PR has an appropriate type label.

Reply:
Ah! Ugh, that's why I was always confused by it...

Reply:
Updated to maybe clearer?